src/node_file.cc

```diff
@@ -1986,8 +1986,26 @@ static void ReadDir(const FunctionCallbackInfo<Value>& args) {
 
   BufferValue path(isolate, args[0]);
   CHECK_NOT_NULL(*path);
+#ifdef _WIN32
+  bool slashCheck = false;
```

Comment:
Please add a comment why this special slash handling is requiered here.

Reply:
Fixed.